src/main/java/xogame/models/Board.java

```diff
@@ -21,10 +21,19 @@ public Board() {
 
     public List<List<Integer>> getRows() {
         List<List<Integer>> rows = new ArrayList<>();
-        
+
         for (int i = 0; i < 3; i++) {
             rows.add(Arrays.asList(board[i * 3], board[i * 3 + 1], board[i * 3 + 2]));
         }
         return rows;
```

Comment:
Can you please add a new line before the return statement in `getRows` and `getColumns`?